### ISSUE TITLE
Improve the list of all datasets

### ISSUE THREAD

**Issue by sseppi** [noi-techpark/it.bz.opendatahub.databrowser#283]
As can you see in the following link, in the _Dataset Overview_ we are now displaying 16 Datasets:
https://databrowser.opendatahub.testingmachine.eu/dataset-overview

In the next iterations we should display in the  _Dataset Overview_ all dataset provided in by the "Dataset MetaData API"
https://tourism.api.opendatahub.bz.it/v1/MetaData

Moreover we should also provide the possibility to the user to see all datasets, also the one that hasn't been configured yet and that aren't retrieved by the "Dataset MetaData API" listed in the:
https://v2-beta.databrowser.opendatahub.com/links

All dataset that aren't present in the MetaData API should be rendered at the bottom of the list including in the information rendered in the box all information that can be retreived by Swagger. The two buttons should redirect as follows:
- Access to table view --> Opens the autoconfig table view
- Show dataset API --> Opens the Swagger or the API call to get all data of the dataset

**Comment by sseppi** [comment]
Hi @gappc ,

I think we rolled back this feature and maybe is active only on your staging environment?

If I try to access the in the testing environment by clicking on ``Show all available Endpoints`` in the Dataset Overview (link below)

https://databrowser.opendatahub.testingmachine.eu/dataset-overview

I land in the following page:

https://databrowser.opendatahub.testingmachine.eu/links

instead updating the list of the dataset.

At this point, if you agree, I propose to link this issue to the improvement of the navigation (#204) and shift this issue in the Milestone 2.3 so we do the changes all in one with the new navigation.

**Comment by gappc** [comment]
@sseppi maybe this was a misunderstanding: all datasets from tourism domain are now shown at the bottom in the [dataset-overview](https://databrowser.opendatahub.testingmachine.eu/dataset-overview) page. We did yet not decide if that's OK or if we need some other mechanisms (need to discuss this in #204 or even better in a new issue).

You can pack the requirement into Milestone 2.3 but given that we are already late in that Milestone and that there are ~13 issues that are not started at all you / we should prioritize what needs to be done next.

In my opinion it would be best if the team sits together and takes a look at the planning in one of the next meetings (or better in a dedicated meeting, because it will take some time).

**Comment by sseppi** [comment]
@MatteoBiasi we put online in the Metadata API the following fields:

* Dataspace (string)
* Category (List)
* DataProvider (List)

We still miss the functionality ``distinct`` in the ``rawfilters`` that will allow you to get all existing values only once.

**Comment by gappc** [comment]
@sseppi isn't `distinct` an aggregation function? not sure how it should work with `rawfilters`

**Comment by sseppi** [comment]
> @sseppi isn't `distinct` an aggregation function? not sure how it should work with `rawfilters`

Rudi is evaluating how to implement it easily and in a clear way, the first proposal was to add this functionality to the ``rawfilter`` or to the ``rawsort``, but it isn't still have bin decided.
